frontend/src/features/properties/map/research/add/AddResearchFileYupSchema.ts

```diff
@@ -2,13 +2,5 @@ import * as Yup from 'yup';
 
 export const AddResearchFileYupSchema = Yup.object().shape({
   name: Yup.string().required('Research File name is required'),
-  properties: Yup.array().of(
-    Yup.object().shape({
-      pid: Yup.string().when('pin', {
-        is: (pin: string) => !!pin,
-        then: Yup.string().nullable(),
-        otherwise: Yup.string().required('valid PID or PIN required'),
-      }),
-    }),
-  ),
+  properties: Yup.array(),
```

Comment:
Is this related to this PR? Doesn't look like...

Reply:
it is, psp-3287 has two issues:
1) research files containing properties without pids or pins can't be saved.
2) research files with any properties throw an error when saving